lib/rules/no-unused-expressions.js

```diff
@@ -11,6 +11,8 @@
 
 module.exports = {
     meta: {
+        type: "problem",
```

Comment:
Let's use the type "suggestion" to be consistent with the original rule

https://github.com/eslint/eslint/blob/master/lib/rules/no-unused-expressions.js#L13

Reply:
Good catch/point. Amended.